docs/integrations/databases/supabase.mdx

```diff
@@ -3,170 +3,666 @@ title: Integrate Supabase with Clerk
+### Add `user_id` to your Supabase tables
 
-<Images 
-  width={3024}
-  height={1652}
-  src="/docs/images/integrations/supabase/template-shortcodes.webp"
-  alt="The 'Create new template' page of the JWT Templates page in the Clerk Dashboard. The page is scrolled down to the 'Claims' section."
-/>
+To show users content scoped to their account, you must create RLS policies that check the user's ID. To access user IDs in your RLS policies, you must add a `user_id` column to the relevant tables:
 
-Supabase requires that JWTs be signed with the H256 signing algorithm and use their signing key. You can locate the JWT secret key in your Supabase project under **Settings --> API** in the *Config* section.
-
-<Images 
-  width={2406}
-  height={1308}
-  src="/docs/images/integrations/supabase/jwt-secret-key.webp"
-  alt="The 'Create new template' page of the JWT Templates page in the Clerk Dashboard. The page is scrolled down to the 'Claims' section."
-/>
-
-Reveal the JWT secret to copy it and then paste it in the **Signing key field** in the Clerk JWT template. After the key is added, you can click the Apply Changes button to save your template.
-
-<Images 
-  width={2392}
-  height={1300}
-  src="/docs/images/integrations/supabase/signing-key.webp"
-  alt="The Supabase JWT template in the Clerk Dashboard. A red box is surrounding the 'Signing Key' section."
-/>
-
-## Configure your client
-
-To configure your client, you need to set some local environment variables. Assuming a React application, set the following.
-
-**Pro tip!** If you are signed into your Clerk Dashboard, you can copy your Clerk publishable key below. Otherwise, you can find it in the Clerk Dashboard on the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page.
-
-<InjectKeys>
-
-```js filename=".env.local"
-NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY={{pub_key}}
-NEXT_PUBLIC_SUPABASE_URL=your-supabase-url
-NEXT_PUBLIC_SUPABASE_KEY=your-supabase-anon-key
-```
-
-</InjectKeys>
-
-You can find your Frontend API value in the Clerk Dashboard on the **[API keys](https://dashboard.clerk.com/last-active?path=api-keys)** page. It will be the **Publishable key**.
-
-<Images
-  width={3024}
-  height={1648}
-  src="/docs/images/integrations/publishable-key.webp"
-  alt="The API keys page in the Clerk Dashboard. There is a red arrow pointing to the copy button next to the 'Publishable key'."
-/>
-
-To get the ones needed for Supabase, navigate to the same **Settings > API page** as before and locate the **anon public key** and **URL**.
-
-<Images
-  width={2392}
-  height={1300}
-  src="/docs/images/integrations/supabase/api-keys.webp"
-  alt="The 'API' page in the Supabase Dashboard. There is a red box surrounding the 'anon public key' and 'URL' fields."
-/>
+- In your Supabase dashboard, in the sidebar, navigate to **Database** > **Tables**. From here, you can choose which table you want to edit, and add the new column.
+    - For this example, you can call the column `user_id`.
```

Comment:
RE: my comment my above I realize you're using _can_ here -- so we are recognizing it's ability to be something else 🤝